app/utils/vr-document-name.js

```diff
@@ -89,7 +89,7 @@ export default class VRDocumentName {
   }
 
   withOtherVersionSuffix(pieceNr) {
-    return `${this.withoutVersionSuffix}${CONFIG.latinAdverbialNumberals[pieceNr].toUpperCase()}`;
+    return `${this.withoutVersionSuffix} ${CONFIG.latinAdverbialNumberals[pieceNr].toUpperCase()}`;
```

Comment:
What if there is no version suffix? Will all piece names get a space behind the name?

Reply:
@tdn we only use this method when adding a new piece to an existing piece. There should always be a next version suffix.